example/android/shared/src/commonMain/kotlin/com/myrnproject/shared/MyComposeView.kt

```diff
@@ -1,15 +1,65 @@
 package com.myrnproject.shared
 
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.Button
 import androidx.compose.material.Text
+import androidx.compose.material.TextField
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.collectAsState
 import androidx.compose.runtime.getValue
+import de.voize.reaktnativetoolkit.annotation.ReactNativeProp
 import de.voize.reaktnativetoolkit.annotation.ReactNativeViewManager
+import kotlinx.coroutines.flow.Flow
 
 @Composable
 @ReactNativeViewManager("MyComposeView")
-internal fun MyComposeView(persistentConfig: PersistentConfig) {
+internal fun MyComposeView(
+    @ReactNativeProp
+    message: Flow<String>,
+    @ReactNativeProp
+    textFieldValue: Flow<String>,
+    @ReactNativeProp
+    boolProp: Flow<Boolean>,
+    @ReactNativeProp
+    intProp: Flow<Int>,
+    @ReactNativeProp
+    doubleProp: Flow<Double>,
+    @ReactNativeProp
+    floatProp: Flow<Float>,
+    @ReactNativeProp
+    onTextFieldValueChange: (String) -> Unit,
+    @ReactNativeProp
+    onButtonPress: () -> Unit,
```

Comment:
Test callbacks with data classes, multiple params, return types and exceptions.

Reply:
data classes and multiple parameters done. return types and exceptions are not possible as these are events